tests/python/pants_test/cache/test_s3_artifact_cache.py

```diff
@@ -0,0 +1,198 @@
+# coding=utf-8
+# Copyright 2014 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import os
+from contextlib import contextmanager
+
+import boto3
+import pytest
```

Comment:
Ah, didn't realize this was an intentional choice in pants. I'm fine with rewriting it (and doing the rest of these changes); will take a stab after work.